docs/installation-docker.md

```diff
@@ -0,0 +1,27 @@
+# Installation
+
+## Requirements
+
+docker + docker-compose
+
+## Run
+
+```bash
+$ cp .env.dist .env
+
+# inside .env file adjust ports and then start containers
+
+$ docker-compose up -d
+```
+
+## Install vendors
+
+```bash
+$ docker run -v `pwd`:/var/www -w /var/www tomhaj/php:7.1-console composer install
```

Comment:
We should use the description and docker images from here: https://github.com/prooph/proophessor-do/blob/master/docs/installation/docker.md#step-2---install-dependencies